content/docs/catalogue/items.md

```diff
@@ -97,3 +97,26 @@ There are two tabs in the bottom part of the screen:
 
 This tab lists the Master Lists associated with this item.
 ![Item detail headers](/docs/catalogue/images/cat_itemmasterlists.png)
+
+
+#### Pack Variants
+
+Please refer to [Item Pack Variants](/docs/catalogue/pack-variants) section for more detail.
+
+You can configure and view item pack variants in this tab. 
+
+![pack variant tab](/docs/catalogue/images/pack_variant_tab.png)
+
+New pack variants are added via **New Pack Variant** button, existing pack variants are edited by clicking on the table row. Deletion can be done via editing modal or by selecting lines via checkboxes and using the 'Select' drop down option.
```

Comment:
```suggestion
New pack variants are added via the **New Pack Variant** button, existing pack variants are edited by clicking on the table row. Deletion can be done via the editing modal or by selecting the checkboxes and using the 'Select' drop down option.
```

Can each of these steps be broken down with screenshots that either have arrows pointed to the `New Pack Variant` button or the delete button?